specification/security/resource-manager/readme.md

```diff
@@ -103,6 +103,35 @@ override-info:
   title: SecurityCenter
 ```
 
+### Tag: package-composite-v2
+
+These settings apply only when `--tag=package-composite-v3` is specified on the command line.
+
+```yaml $(tag) == 'package-composite-v2'
+input-file:
+- Microsoft.Security/stable/2018-06-01/pricings.json

```

Comment:
I think we should sort by api-version date like [sql did here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md)